packages/palette_generator/pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: palette_generator
 description: Flutter package for generating palette colors from a source image.
 repository: https://github.com/flutter/packages/tree/main/packages/palette_generator
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+palette_generator%22
-version: 0.3.3
+version: 0.3.3+1
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.14.0 <3.0.0"
   flutter: ">=1.15.21"
```

Comment:
Where there is already a Flutter constraint, it should be updated to 3.0.

Reply:
oh, i didn't know about that, done.